usaon_vta_survey/models/tables.py

```diff
@@ -64,6 +64,26 @@ def __io__(cls) -> IORelationship:
         return io
 
 
+class ResponseObjectFieldMixin:
+    """Provide shared fields between all relationshio objects to reduce repition."""
+
+    short_name = Column(String(256), nullable=False)
+    full_name = Column(String(256), nullable=False)
+    organization = Column(String(256), nullable=False)
+    # should this be funding agency?
+    funder = Column(String(256), nullable=False)
```

Comment:
@hazelshapiro

Reply:
The "funder" would refer to a funding agency (or another group if it's not federally funded), while "organization" refers to the group that manages and runs the system. In some cases, they will be the same.